nodehid.js

```diff
@@ -31,11 +31,19 @@ function HID() {
     /* Now we have `this._raw` Object from which we need to
         inherit.  So, one solution is to simply copy all
         prototype methods over to `this` and binding them to
-        `this._raw`
+        `this._raw` if we don't already have a wrapper implemented
     */
-    for(var i in binding.HID.prototype)
-        if(i != "close" && i != "read")
-            this[i] = binding.HID.prototype[i].bind(this._raw);
+    for (var i in binding.HID.prototype) {
+		if (!HID.prototype[i]) {
+			this[i] = (function(fn){
```

Comment:
@todbot An advantage of the test on line 37 is it uses the existence of a method on the javascript prototype as an indicator for whether it should attach the native method to the constructed object - meaning you don't have to keep adding exclusions as you add facades for methods (as seen with 'open' in https://github.com/node-hid/node-hid/pull/178)